cluster/src/main/java/org/apache/iotdb/cluster/utils/ClusterNode.java

```diff
@@ -73,6 +75,9 @@ public String toString() {
         + dataPort
         + ", clientPort="
         + clientPort
+        + ", clientIp='"
+        + clientIp
+        + '\''
```

Comment:
what's this?

Reply:
clientIp is the `rpc_address` in iotdb-engine.properties which is used to communicate with the client.

Reply:
I mean `'\'`

Reply:
got it. I'll fix this